### DISCUSSION TITLE
Scale widgets within container

### DISCUSSION THREAD

**Discussion by p4ymak** [emilk/egui#1764] [Q&A]
Hello! Is it possible to scale widgets like it is done by `set_pixels_per_point`, but limited to a certain container?

**Comment by zicklag** [comment]
Hey I need exactly the same thing!

The issue is that I have a game in retro-style scale, and I'm using Egui for the in-game UI elements. But I also have some debugging windows with diagnostic info that end up _way_ to big at the retro scale, so I would like those windows to use a different scale.

Unfortunately, for all I've seen in Egui, there isn't a way to do it yet, and I'm not sure that it would be easy to accomplish either.

**Comment by zicklag** [comment]
Related: https://github.com/emilk/egui/discussions/261.
